ui/components.go

```diff
@@ -17,162 +17,17 @@ const (
 var (
 	// layout.Flex: Vertical
 	vertFlex = layout.Flex{Axis: layout.Vertical}
-	// layout.Flex: Vertical, SpaceBetween
-	vertFlexSB = layout.Flex{Axis: layout.Vertical, Spacing: layout.SpaceBetween}
 	// layout.Flex: Horizontal
 	horFlex = layout.Flex{}
-	// layout.Flex: Horizontal
-	horFlexSB = layout.Flex{Spacing: layout.SpaceBetween}
-
 	// layout.Rigid
 	rigid = layout.Rigid
-	// layout.Flexed
-	flexed = layout.Flexed
 )
```

Comment:
These can be deleted and called directly from the individual pages where they are used.